android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -528,28 +529,18 @@ private void maybeShowPendingTransactions() {
         }
     }
 
-    private void maybeShowSignTxRequestLayout() {
-        assert mBraveWalletService != null;
-        mBraveWalletService.getPendingSignTransactionRequests(requests -> {
-            if (requests != null && requests.length != 0) {
-                openBraveWalletDAppsActivity(
-                        BraveWalletDAppsActivity.ActivityType.SIGN_TRANSACTION);
-                return;
-            }
-            maybeShowSignAllTxRequestLayout();
-        });
-    }
-
-    private void maybeShowSignAllTxRequestLayout() {
+    // TODO(apaymyshev): refactor this to have a better name.
+    private void maybeShowSignSolTransactionsRequestLayout() {
```

Comment:
I'm going to take care of this in my next PR, entangle the chains and and detach the logic that is not related to SOL transactions!